[OSDOCS-6403]: Add warning on cluster deletion docs (#62221)
openshift/openshift-docs: modules/rosa-deleting-cluster.adoc

After deleting the cluster, you can clean up the cluster-specific Identity and Access Management (IAM) resources in your AWS account by using the ROSA CLI (`rosa`). The cluster-specific resources include the Operator roles and the OpenID Connect (OIDC) provider.

[NOTE]
====
The cluster deletion must complete before you remove the IAM resources, because the resources are used in the cluster deletion and clean-up processes.
====

If add-ons are installed, the cluster deletion takes longer because add-ons are uninstalled before the cluster is deleted. The amount of time depends on the number and size of the add-ons.
endif::sts[]

[IMPORTANT]
====

> If the cluster that created the VPC during the installation is deleted, the associated installer-created VPC will also be deleted, resulting in the failure of all the clusters that are using the same VPC. Additionally, any resources created with the same tagSet key-value pair of the installer-created resources and labeled with a value of `owned` will also be deleted.

Reviewer: ```suggestion
If the cluster that created the VPC during the installation is deleted, the associated installer-created VPC will also be deleted, resulting in the failure of all the clusters that are using the same VPC. Additionally, any resources created with the same `tagSet` key-value pair of the installer-created resources and labeled with a value of `owned` will also be deleted.
```

Reviewer: If this is a specific object reference it needs to be in back ticks or if it is a generic it needs to be tag set. I agree with @tmalove and think `installer-created` needs to be addressed per [redhat-documentation.github.io/supplementary-style-guide/#_i](https://redhat-documentation.github.io/supplementary-style-guide/#_i) please.

Author: > If this is a specific object reference it needs to be in back ticks or if it is a generic it needs to be tag set. I agree with @tmalove and think `installer-created` needs to be addressed per [redhat-documentation.github.io/supplementary-style-guide/#_i](https://redhat-documentation.github.io/supplementary-style-guide/#_i) please.
Hello @JoeAldinger,
    Thanks for taking a look at this doc. I will check out the style guide and let you know if I have any questions about what is stated. FWIW, I had actually pulled that text from an existing doc. Regardless, let me take a look at the style guide and make sure I understand it. Thanks!  
![image](https://github.com/openshift/openshift-docs/assets/122639474/e9a3370e-1641-47d0-83f7-f1487d06d4af)

Author: > If this is a specific object reference it needs to be in back ticks or if it is a generic it needs to be tag set. I agree with @tmalove and think `installer-created` needs to be addressed per [redhat-documentation.github.io/supplementary-style-guide/#_i](https://redhat-documentation.github.io/supplementary-style-guide/#_i) please.

...and yes, going back over my notes with the SRE, it is specific and should have back ticks. Will update.

Author: Hello @JoeAldinger.
     I believe all the requested changes to be completed in order to merge are now complete. Thanks!

> If the cluster that created the VPC during the installation is deleted, the associated installer-created VPC will also be deleted, resulting in the failure of all the clusters that are using the same VPC. Additionally, any resources created with the same tagset key/value pair of the installer-created resources and labeled with a value of `owned` will be deleted as well.

Reviewer 2: ```suggestion
If the cluster that created the VPC during the installation is deleted, the associated installation program-created VPC will also be deleted, resulting in the failure of all the clusters that are using the same VPC. Additionally, any resources created with the same tagset key-value pair of the resources created by the installation program and labeled with a value of `owned` will also be deleted.
```
